Rakefile

```diff
@@ -0,0 +1,30 @@
+require 'open3'
+
+task default: :test
+
+task test: 'dummy:setup' do
+  sh 'npx playwright test --reporter=dot'
+end
+
+task test_ui: 'dummy:setup' do
+  sh 'npx playwright test --ui'
+end
+
+namespace :dummy do
+  desc 'Setup the dummy application'
+  task :setup do
+    Dir.chdir('test/dummy') do
+      rm_rf 'node_modules'
```

Comment:
Why do we need to remove node_modules?

Reply:
I think that means every time someone runs `rake test` they'll have to delete and re-download node_modules

Reply:
@justwiebe unfortunately, because of how yarn/npm pack dependencies, even though turbo-confirm is a local/relative dependency.  It's still necessary to destroy & repack in order to pick up changes within the dummy app.  I experimented with rake file tasks, attempting to only run this task if something within the /src directory changed.  But it wasn't straightforward and I didn't want to spend a lot of time on it.

Now that you bring it up though, I suppose it's only necessary to remove the @rolemodel/turbo-confirm directory.  Which would speed this up quite a bit..

Reply:
@justwiebe Or if you know of a way to force yarn not to cache it?  Yarn does have a `link` command, but that doesn't seem to effect the node_modules caching, only how yarn finds the source..

Reply:
@justwiebe turns out you can do that afterall! I'll ping you after I update this, thanks for reviewing this!